dz12_ex1.py

```diff
@@ -0,0 +1,45 @@
+'''1. Використати файл як базу даних для збереження записів телефонної книги із попередніх завдань.
+
+Ваша телефонна книга, що до цього містилася в dict, має зберігатися у вигляді тексту в JSON форматі.
+При закритті програми і повторному відкритті всі попередні дані мають бути доступними.
+Підказка: Ви можете використати бібліотеку json, яка допоможе із перетворенням dict в JSON string (при записі в файл) і JSON string в dict (при читанні із файлу). Файл слід оновлювати після кожної успішної операції add або delete.
+'''
+import json
+
+with open('telbook.json') as dfile:
```

Comment:
Check here if file exists (you can use try except) and init the `telefon_book` as an empty dict if it doesn't